### PR TITLE
XIVY-12288 only show class name of type in table

### PR DESCRIPTION
**Pull request by ivy-lgi**
Fully qualified name can still be seen in the detail view and is provided as a tooltip in the table.

image

### CODE REVIEW

File: packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css [hunk 1]
```diff
@@ -2,3 +2,11 @@
...
   min-height: 0;
 }
+
+.cell-with-tooltip {
+  display: inline-block;
+}
+
+td .ui-tooltip-content {
+  color: var(--body);
+}
```
Comment (ivy-lli) [inline packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css:12]: why are these needed?

Comment (ivy-lgi) [reply packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css:12]: `display: inline-block` adjusts the width of the cell content to center the tooltip. Without that, the tooltip looks like this:

image

`color: var(--body)` overwrites the tooltip color. Otherwise, it seems to take the color from the cell, which when selected results in the tooltip content not being visible:

image

Comment (ivy-lli) [reply packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css:12]: For the first I think you can simply use a <span> instead of a <div>.
For the second, I think this should be improved in the ui-components directly

**Review by ivy-lli — APPROVED** [review]
See comment for style stuff, afterwards good 👍

File: packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css (hunk 1 above)
Comment (ivy-lgi) [reply packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css]: Going to make the tooltip improvement in ui-components.
